src/manager/FileBackedTaskManager.java

```diff
@@ -30,15 +31,9 @@ private void save() {
             }
 
             List<String> taskCSVLinesArray = new ArrayList<>();
-            for (Task task : taskHashMap.values()) {
-                taskCSVLinesArray.add(task.toCSVLine());
-            }
-            for (Epic epic : epicHashMap.values()) {
-                taskCSVLinesArray.add(epic.toCSVLine());
-            }
-            for (SubTask subTask : subTaskHashMap.values()) {
-                taskCSVLinesArray.add(subTask.toCSVLine());
-            }
+            taskHashMap.values().stream().forEach(task -> taskCSVLinesArray.add(task.toCSVLine()));
```

Comment:
Стримы принято разбивать по строкам, чтобы улучшить читаемость кода
```suggestion
            taskHashMap.values().stream()
                .forEach(task -> taskCSVLinesArray.add(task.toCSVLine()));
```
Поправь, пожалуйста, все стримы в данном классе